user_guide/configuration/printing_paths.rst

```diff
@@ -0,0 +1,50 @@
+Printing paths
+==============
+
+When showing test results or failures Behat will print the path to the related files. Usually it will
+simply print a path which is relative to the current directory but there are some configuration options
+that will let you modify this:
```

Comment:
I think "By default" is a little clearer than "Usually" - if working in a shared project
then "usual" will depend on whether someone has already put these options in the
config file?

```suggestion
When showing test results or failures Behat will print the path to the related files. By default 
the path will be relative to the current directory. This behaviour can be modified through 
configuration or on the command line:
```